habitat/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+export GEM_HOME="$(hab pkg path chef/chef-client)/ruby/2.3.0"
```

Comment:
In playing around with this branch, we had to make the following change for the run hook to use the package built under my origin.

```diff
-export GEM_HOME="$(hab pkg path chef/chef-client)/ruby/2.3.0"
+export GEM_HOME="{{pkg.path}}/ruby/2.3.0"
```